StripeIdentity/StripeIdentity/Source/Elements/IndividualFormElement.swift

```diff
@@ -99,6 +107,12 @@ final class IndividualFormElement: ContainerElement {
 
         elements = [Element]()
 
+        if missing.contains(.phoneNumber) {
+            phoneNumberSectionElement = elementsFactory.makePhoneSection(countries: Array(individualContent.phoneNumberCountries.keys))
+            elements.append(phoneNumberSectionElement!)
+        } else {
+            phoneNumberSectionElement = nil
```

Comment:
should `phoneNumberSectionElement` also be removed from `elements` array here?

Reply:
`elements[]` was initialized empty and got added conditionally, so `phoneNumberSectionElement` here was never in the `elements`